src/features/device-state/index.ts

```diff
@@ -56,7 +56,10 @@ export const setup = (app: Application) => {
 };
 
 export interface Events {
-	'get-state': (deviceId: number, req: Pick<Request, 'apiKey'>) => void;
+	'get-state': (
+		deviceId: number,
+		req: Pick<Request, 'apiKey'> & { config?: Dictionary<string> },
+	) => void;
```

Comment:
You can also add extra args, you don't have to bundle it into one. It doesn't matter either way though